definitions/npm/typescript_v3.3.x/test_typescript_v3.3.x.js

```diff
@@ -0,0 +1,3 @@
+// @flow
+
+import type { LanguageServiceHost } from 'typescript';
```

Comment:
Previously, this would fail with the following under Flow 0.154.0 and later:

```
Cannot extend `GetEffectiveTypeRootsHost` [1] with `LanguageServiceHost` because function type [2] cannot be unbound
    from the context [2] where it was defined in the `this` parameter of property `getCurrentDirectory`. [method-unbinding]
```